src/content/es/fundamentals/codelabs/your-first-pwapp/index.md

```diff
@@ -846,68 +852,70 @@ Además de cualquier IU que agregue para instalar su aplicación, los usuarios t
-
-Tenga en cuenta que si está ejecutando en el escritorio desde localhost, su PWA instalado puede mostrar una pancarta de dirección porque localhost no se considera un host seguro.
+A continuación, nos aseguramos de que todo se instala correctamente y de que nuestros eventos se activan correctamente. Puedes hacerlo tanto en tu escritorio como en tu móvil. Si deseas probar esto en el móvil, asegúrate de que estás utilizando la depuración remota para poder ver lo que se está registrado en la consola.
+
+1. Abre Chrome y, en una nueva pestaña del navegador, navega a tu PWA Clima.
+2. Abre DevTools y cambia al panel de la consola.
+3. Haz clic en el botón de instalación en la esquina superior derecha.
+▢ Verifica que el botón de instalación desaparezca
+▢ Verifica que se muestra el cuadro de diálogo modal de instalación.
+4. Haz clic en Cancelar.
+▢ Verifica que &quot;*El usuario rechazó la solicitud A2HS*&quot; se muestra en la salida de la consola.
+▢ Verifica que el botón de instalación vuelva a aparecer.
+5. Haz clic en el botón Instalar nuevamente, luego haz clic en el botón Instalar en el diálogo modal.
+▢ Verifica que &quot;*El usuario aceptó la solicitud A2HS*&quot; se muestra en la salida de la consola.
+▢ Verifica que &quot;*aplicación meteorológica se haya instalado*&quot; se muestre en la salida de la consola.
+▢ Verifica que la aplicación meteorológica se agrega al lugar donde normalmente encontrará las aplicaciones.
+6. Inicia la PWA Clima.
+▢ Verifica que la aplicación se abre como una aplicación independiente, ya sea en una ventana de la aplicación en el escritorio o en pantalla completa en el móvil.
+
+Tenga en cuenta que si está ejecutando en el escritorio desde localhost, tu PWA instalada puede mostrar una pancarta de dirección porque localhost no se considera un host seguro.
 
 #### Verifica que la instalación de iOS funcione correctamente
 
 Veamos también el comportamiento en iOS. Si tiene un dispositivo iOS, puede usarlo, o si está usando una Mac, pruebe el simulador de iOS disponible con Xcode.
 
-1. Abra Safari y en una nueva pestaña del navegador, navegue a su Weather PWA.
-2. Haga clic en el *Compartir*! Botón [8ac92dd483c689d3.png](img/8ac92dd483c689d3.png) .
-3. Desplácese hacia la derecha y haga clic en el botón *Agregar a la pantalla de inicio*.
-▢ Verifique que el título, la URL y el icono sean correctos.
-4. Haga clic en *Agregar.*
-▢ Verifique que el icono de la aplicación se agregue a la pantalla de inicio.
-5. Inicie el Weather PWA desde la pantalla de inicio.
-▢ Verifique que la aplicación se inicie en pantalla completa.
+1. Abre Safari y en una nueva pestaña del navegador, navegue a tu PWA Clima.
+2. Haz clic en el *Compartir*! Botón [8ac92dd483c689d3.png](img/8ac92dd483c689d3.png) .
+3. Desplácese hacia la derecha y Haz clic en el botón *Agregar a la pantalla de inicio*.
```

Comment:
`Desplázate..`  to keep the conjugation in the second person.
...y haz clic en...